tcms/testruns/templates/testruns/get.html

```diff
@@ -78,26 +71,33 @@ <h2 class="card-pf-title" style="text-align: left">
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
-                    {{ object.start_date }}
-                </h2>
+                    {% if object.start_date == None %}
+                        <button class="btn btn-success btn-xs" type="button" id="start-button"
+                                data-toggle="tooltip" data-placement="top" title="{% trans 'Start' %}">
+                            <i class="fa fa-check" style="color: white; margin: auto; "></i>
+                        </button>
+                    {% endif %}
+                        <span class="start-date">
+                            {{ object.start_date|default:'-' }}
+                        </span>
 
+                </h2>
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned stop' %}:
                     {{ object.planned_stop|default:'-' }}
                 </h2>
-
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa-calendar-check-o"></span>{% trans 'Finished at' %}:
+                    {% if object.stop_date == None %}
```

Comment:
The canonical version is `if not object.stop_date` b/c None and other zero values are evaluated as False in boolean context.